py4j-java/src/test/java/py4j/instrumented/InstrumentedClientServerApplication.java

```diff
@@ -60,7 +61,8 @@ public void startServerWithPythonEntry(boolean shutdown) {
 		InstrClientServer server2 = new InstrClientServer(GatewayServer.DEFAULT_PORT + 5,
 				GatewayServer.defaultAddress(), GatewayServer.DEFAULT_PYTHON_PORT + 5, GatewayServer.defaultAddress(),
 				GatewayServer.DEFAULT_CONNECT_TIMEOUT, GatewayServer.DEFAULT_READ_TIMEOUT,
-				ServerSocketFactory.getDefault(), SocketFactory.getDefault(), this, false, true);
+				GatewayServer.DEFAULT_CREATE_SOCKET_CONNECTION_TIMEOUT, ServerSocketFactory.getDefault(),
+				SocketFactory.getDefault(), this, false, true);
```

Comment:
already have `GatewayServer.DEFAULT_CONNECT_TIMEOUT`

```suggestion
				ServerSocketFactory.getDefault(), SocketFactory.getDefault(), this, false, true);
```